### DISCUSSION TITLE
Account restricted

### DISCUSSION THREAD

**Discussion by CrispyToast23** [codewars/codewars.com#2987] [General]
My account TS_Akademie just got suspended and I don’t why, I didn’t copy any solutions of the internet. I will admit that I used ChatGPT to help myself when I didn’t know what was wrong with my code but I didn’t go around copying code from the internet. So could I please just get told what went wrong?

**Comment by Blind4Basics** [comment]
"chatGPT" is what went wrong. Cheers

**Reply by CrispyToast23** [reply]
I never read anything about it being banned tho

**Reply by Blind4Basics** [reply]
Your account isn't banned, it's locked.
The CoC is talking about using your _own_ code. You didn't write that code, so it's a match for restrictions or invalidations.

**Reply by CrispyToast23** [reply]
So if I would take functions from StackOverflow and use them in my code that would also be not allowed, because that’s what I basically did.

**Reply by Blind4Basics** [reply]
yes. Or from a gitxxx repo, or...

**Reply by Mythrixx** [reply]
That's absurd! Anything that makes writing your code more efficient should be allowed.

**Reply by Kacarott** [reply]
But you didnt write it, its not "your code", thats the point?
